sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -465,6 +465,20 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
       s"""A method named "$name" is not declared in any enclosing class nor any supertype""")
   }
 
+  def methodNotFoundError(
```

Comment:
this was defined as `internalError` previously, really need to add a new `errorClass`?

Reply:
I searched the codebase, seems `Invoke` `StaticInvoke` is only used internally, let me convert it to `internalError` as you suggested.

Reply:
updated